config/manager/manager.yaml

```diff
@@ -30,6 +31,7 @@ spec:
           - "--audit-interval=30"
           - "--port=8443"
           - "--logtostderr"
+          - "--exempt-namespace=gatekeeper-system"
```

Comment:
In addition to `gatekeeper-system`, I think we should also add `kube-system` to the default exemption list. I know there are users who have the `control-plane` label on `kube-system` today to ensure resources in that namespace are not impacted by Gatekeeper.

Reply:
I'm more concerned about unexpectedly opening a security hole for users who haven't already done so.

Users that have set the control-plane label on kube-system will break anyway once the label becomes deprecated. Because we should not modify people's kube-system namespace by default including kube-system as an excused label will be the worst of both worlds: previous exclusion efforts will still fail and users who didn't want to exclude kube-system will unknowingly be vulnerable to people adding the exclusion label.

This was discussed in the meeting 2.5 weeks ago.